recipes/label_parsing.ipynb

```diff
@@ -0,0 +1,2944 @@
+{
```

Comment:
<p>This should be the main entrypoint to the parsed label class, so this section should move up, before detailing what a parsed label is and how to use it. Converting from an existing dict should come here. WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
done